packages/ensnode-utils/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
```

Comment:
we should leave the indexer's tsconfig as ponder has generated it, but if we ever have more than one other package using a similar tsconfig we should do the monorepo thing where we add a `tsconfig` package that other projects can depend on an inherit from.